resources/views/layouts/adminLayout.edge

```diff
@@ -1,143 +1,143 @@
-          <a class='dropdown-item' href='register.html'>Search Bookings</a>
+  <head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <meta name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no'>
+    <meta name='description' content=''>
+    <meta name='author' content=''>
+
+    @!section('extracss')
+
+    <title>
+      @!section('title')
+    </title>
+
+    <!-- Custom fonts for this template-->
+    <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.7.1/css/all.css' integrity='sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr' crossorigin='anonymous'>
+    <link href='https://fonts.googleapis.com/css?family=Lato:300,400,700|Roboto:300,400,700' rel='stylesheet'>
+    
+    <!-- Page level plugin CSS-->
+    <link href='vendor/datatables/dataTables.bootstrap4.css' rel='stylesheet'>
+
+    <!-- Custom styles for this template-->
+    {{ style('css/adminDash') }}
+
+  </head>
+
+  <body id='page-top'>
+    {{--  Left side navbar, with logo  --}}
+    <nav class='navbar navbar-expand navbar-dark bg-dark static-top'>
+      {{-- <a class='navbar-brand mr-1' href='index.html'><img src='door_logo.png'></img></a> --}}
+      <a class='navbar-brand mr-1' href='{{route('home')}}'><img src='door_logo.png'></a>
+      <button class='btn btn-link btn-sm text-white order-1 order-sm-0' id='sidebarToggle' href='#'>
+        <i class='fas fa-bars'></i>
+      </button>
+
+      <!-- Navbar -->
+      <ul class='navbar-nav ml-auto'>
+        <li class='nav-item dropdown no-arrow'>
+          <a class='nav-link dropdown-toggle' href='#' id='userDropdown' role='button' data-toggle='dropdown' aria-haspopup='true' aria-expanded='false'>
+            {{--  Displays user name  --}}
+            {{auth.user.username}}
+            <i class='fas fa-user-circle fa-fw ml-2'></i>
+          </a>
+          <div class='dropdown-menu dropdown-menu-right' aria-labelledby='userDropdown'>
+            <a class='dropdown-item' href='#'>Settings</a>
+            <a class='dropdown-item' href='#'>Activity Log</a>
+            <div class='dropdown-divider'></div>
+            {{--  Logout option that openss a modal to confirm  --}}
+            <a class='dropdown-item' href='#' data-toggle='modal' data-target='#logoutModal'>Logout</a>
+          </div>
+        </li>
+      </ul>
+    </nav>
+
+    <div id='wrapper'>
+      <!-- Sidebar -->
+      <ul class="sidebar navbar-nav shadow">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" id="pagesDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <i class="fas fa-fw fa-folder"></i>
+            <span>Rooms</span>
+          </a>
+          <div class="dropdown-menu" aria-labelledby="pagesDropdown">
+            <a class="dropdown-item" href="login.html">Add Room</a>
```

Comment:
`<a class='dropdown-item' href='{{ route('addRoom')}}'>Add Room</a>
          <a class='dropdown-item' href='{{ route('removeRoom')}}'>Remove Room</a>`  

Need these routes for add and remove forms